release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -235,6 +235,17 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-dev-exp"]
 === Developer experience
 
+[id="ocp-4-8-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-8-auth-tp-aws-sts"]
+==== Running {product-title} using AWS Security Token Service (STS) for credentials is generally available
+
+You can now use the Cloud Credential Operator (CCO) utility (`ccoctl`) to configure the CCO to use the Amazon Web Services Security Token Service (AWS STS). When the CCO is configured to use STS, it assigns components IAM roles that provide short-term, limited-privilege security credentials.
+
+This feature was previously introduced as a Technology Preview feature in {product-title} 4.7, and is now generally available in {product-title} 4.8.
+
+For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-mode-sts[Support for Amazon Web Services Secure Token Service (AWS STS)].
```

Comment:
The link text title doesn't match the assembly title. I suspect that's because you are looking for the Release Note to be a bit more generic than saying "manual mode"?

Reply:
Yeah, but I also take your point. Will tweak this a bit to satisfy both aspects. TY!